packages/language/src/validation/pli-validator.ts

```diff
@@ -70,6 +77,36 @@ export class Pl1Validator {
     }
   }
 
+  checkDeclareStatement(
+    node: AST.DeclareStatement,
+    accept: PliValidationAcceptor,
+  ): void {
+    const checkNode = (hasLevel: boolean) => (child: AST.SyntaxNode) => {
+      if (child.kind === AST.SyntaxKind.DeclaredItem) {
+        if (hasLevel && child.level) {
```

Comment:
Minor: Due to a quirk in JS, `child.level` can evaluate to `false` if it is `0`. We should probably just check for `levelToken`. In that case, we also don't need to do `levelToken!`.